src/utils/Dom.ts

```diff
@@ -560,13 +569,20 @@ export class Dom {
   public trigger(type: string, data?: { [key: string]: any }): void {
     const modifiedType = this.processEventTypeToBeJQueryCompatible(type);
     const jq = JQueryUtils.getJQuery();
-    if (jq) {
+    if (jq && !Dom.forceNativeEvent) {
       jq(this.el).trigger(modifiedType, data);
-    } else if (CustomEvent !== undefined) {
+    } else if (window['CustomEvent'] !== undefined) {
       const event = new CustomEvent(modifiedType, { detail: data, bubbles: true });
       this.el.dispatchEvent(event);
     } else {
-      new Logger(this).error('CANNOT TRIGGER EVENT FOR OLDER BROWSER');
+      try {
+        // IE11 compatibility
```

Comment:
I would remove this comment, and extract the three lines into a method called `triggerIE11CompatibleEvent(modifiedType, data) {}`.

Reply:
👍